File: Multiples 3&5/1.rb
count = 0 

(1...1000).each do |i|

	if i % 3 == 0 && i % 5 == 0
		count += i
	elsif i % 5 == 0 
		count += i
	elsif i % 3 == 0
		count += i
	end		
end

puts "Answer: #{count}"

File: 1/1.js
var count = 0 

for (var i = 0; i < 1000; i++) {
	if (i % 3 == 0 && i % 5 ==0) {
		count += i;
	}
	else if(i % 5 == 0){
		count += i;
	}
	else if(i % 3 == 0){
		count += i
	}
};

console.log(count);